proto/acl_state.proto

```diff
@@ -16,5 +16,11 @@ message ACLState {
 
     // count of packets hitting the rule
     int32 packet_count = 3;
+
+    // number of times the packet count increases
+    int32 interval_packet_count = 4;
```

Comment:
this isn't a packet count

Reply:
> Code looks ok, but I think we need better names for the fields of the exposed api. Take a moment to think about how they all fit together and consider them as variants of the same thing... on one end there's "description" and the other is "interval_packet_count_last_change" -- which goes from 1 thing to 5 things.

Do you mean the names of the fields should have similar length?

Reply:
> this isn't a packet count

What about inreasing_interval_count?